site/src/pages/RoadmapPage/Quarter.tsx

```diff
@@ -150,6 +151,21 @@ const Quarter: FC<QuarterProps> = ({ year, yearIndex, quarterIndex, data }) => {
           );
         }}
       </StrictModeDroppable>
+
+      {isMobile && (
+        <>
+          <Button
+            variant="light"
```

Comment:
Make this dark mode compatible by updating variant reactively to theme.

```suggestion
            variant={buttonVariant}
```